ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetBulk.java

```diff
@@ -149,27 +154,26 @@ public Ds3Object apply(final Contents input) {
 
         final Ds3ClientHelpers.Job job = helper.startReadJob(this.bucketName, objects,
                 ReadJobOptions.create()
-                .withPriority(this.priority));
+                        .withPriority(this.priority));
 
         job.transfer(new LoggingFileObjectGetter(getter));
 
         if (sync) {
             if (prefix != null) {
                 return "SUCCESS: Synced all the objects that start with '" + this.prefix + "' from " + this.bucketName + " to " + this.outputPath.toString();
-            }
-            else {
+            } else {
                 return "SUCCESS: Synced all the objects from " + this.bucketName + " to " + this.outputPath.toString();
             }
         }
 
-        return "SUCCESS: Wrote all the objects that start with '"+ this.prefix +"' from " + this.bucketName + " to " + this.outputPath.toString();
+        return "SUCCESS: Wrote all the objects that start with '" + this.prefix + "' from " + this.bucketName + " to " + this.outputPath.toString();
     }
 
     private String restoreAll(final Ds3ClientHelpers.ObjectChannelBuilder getter) throws XmlProcessingException, SignatureException, IOException, SSLSetupException {
         final Ds3ClientHelpers helper = getClientHelpers();
         final Ds3ClientHelpers.Job job = helper.startReadAllJob(this.bucketName,
                 ReadJobOptions.create()
-                .withPriority(this.priority));
+                        .withPriority(this.priority));
```

Comment:
I think this indentation is extra - double indent (8 spaces) indicates continuation from the previous line, this is triple indented.